tests/acceptance/features/apiSpaces/listSpaces.feature

```diff
@@ -131,3 +131,43 @@ Feature: List and create spaces
       | name             | Project Venus                    |
       | quota@@@total    | 2000                             |
       | root@@@webDavUrl | %base_url%/dav/spaces/%space_id% |
+
+  
+  Scenario: A project space owner should always be one of the space managers
+    Given the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created a space "Project Venus" with the default quota using the GraphApi
+    And user "Alice" has shared a space "Project Venus" to user "Brian" with role "manager"
+    When user "Alice" lists all available spaces via the GraphApi with query "$filter=driveType eq 'project'"
+    Then the json responded should contain a space "Project Venus" owned by "Alice" with these key and value pairs:
```

Comment:
fragile part, we cannot guarantee that `Alice` is picked as owner, it COULD be `Brian` too.... no guaranteed pick order.

Reply:
true, thanks. If we have 2 managers any, of them can be the owner
I changed test